apps/sms/desktop-mock/navigator_moz_mobilemessage.js

```diff
@@ -1056,7 +1057,7 @@
     var completeSend = function() {
       request.result = sendInfo;
 
-      if (simulation.failState()) {
+      if (true) {
         sendInfo.message.delivery = 'error';
         request.error = {
           name: window.MessagesDebugError
```

Comment:
for the final patch, you'll need to revert to the initial version for this file; it was useful to test the code but we can't leave it like this :)